components/add_groups_to_team_modal/add_groups_to_team_modal.jsx

```diff
@@ -0,0 +1,278 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
```

Comment:
This component is a fairly straight copy of components/add_users_to_team/add_users_to_team.jsx.